docs/install.md

```diff
@@ -1215,8 +1223,12 @@ The current API for PMEM-CSI `Deployment` resources is:
 | pullPolicy | string | Docker image pull policy. either one of `Always`, `Never`, `IfNotPresent` | `IfNotPresent` |
 | logLevel | integer | PMEM-CSI driver logging level | 3 |
 | deviceMode | string | Device management mode to use. Supports one of `lvm` or `direct` | `lvm`
-| controllerResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for controller pod |
-| nodeResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for the pods running on node(s) |
+| controllerResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for controller pod [**_Deprectead_**] |
```

Comment:
What does "deprecated" mean here? Isn't `controllerResources` gone in v1beta1?

We probably need a more nuanced way of describing the API:
- for `apiVersion`, list v1beta1 and v1alpha1 as valid values
- add a general remark outside of the table that fields are valid in all API revisions unless noted otherwise
- for fields that only exist in some revision(s), list those revisions in the description

Reply:
> What does "deprecated" mean here? Isn't `controllerResources` gone in v1beta1?
> 
Yes, `v1beta1` uses `controllerDriverResources` and `provisionerResources` in place of `controllerResources`.

> We probably need a more nuanced way of describing the API:
> 
> * for `apiVersion`, list v1beta1 and v1alpha1 as valid values
> * add a general remark outside of the table that fields are valid in all API revisions unless noted otherwise
> * for fields that only exist in some revision(s), list those revisions in the description

Made changes as suggested.